src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Version>0.4</Version>
+    <Version>0.5.0</Version>
```

Comment:
Does this really need a version bump this big?  Why not 0.4.1?

Reply:
I felt like all of the projects getting bumped to net6.0 and many of the dependencies causing a lot of version bumps in other dependencies, potentially bumped it to a minor version update. Though maybe this is contingent on PSRule updates also making it in. But open to comment -- @VeraBE and @anaismiller any thoughts?

Reply:
The targeting change to internal libraries doesn't really change the behavior overall of the tool, so I don't see that being important here.  Same with dependencies.  PSRule update seems like just a patch version to me.

Reply:
For the bump to net6.0, would that impact users of our package? If so, then I would do a major update. Otherwise, I would not. And agree with Johnathon that the other updates are all minor changes.

Reply:
Actually this new release will include all the work to handle Bicep modules, right?